pom.xml

```diff
@@ -1064,6 +1059,22 @@
   </build>
 
     <profiles>
+        <profile>
+            <id>enable-jacoco</id>
```

Comment:
I think this is already defined in the parent POM, see https://github.com/jenkinsci/plugin-pom/blob/8c269dbd6f2fd7810efd7ec8df9f6bdba5b37c82/pom.xml#L1059, so I don't think you need it here unless you have special overrides. If you do override it here, you probably want to copy https://github.com/jenkinsci/plugin-pom/blob/8c269dbd6f2fd7810efd7ec8df9f6bdba5b37c82/pom.xml#L1080 to exclude generated classes.

Reply:
good call.